app/download/views.py

```diff
@@ -13,14 +13,15 @@ def download_document(service_id, document_id):
         return jsonify(error='Missing decryption key'), 400
 
     filename = request.args.get('filename')
+    sending_method = request.args.get('sending_method')
```

Comment:
Can remove the `.get` now that args are required? Make sure we fail loudly if those are not here

Reply:
sure